docs/src/components/base/layout/README.md

```diff
@@ -0,0 +1,43 @@
+# Layout
+## Float layout
+
+The float layout grid uses a `928px` max-width. Grid columns collapse down to a single width on small screens.
+
+
+### Float layout grid rows
+
+A `.grid` represents a single row. For multiple rows, use multiple grids.
+
+```html
+<div class="grid">
+  <div class="grid-width-4"></div>
+  <div class="grid-width-8"></div>
+</div>
+<div class="grid">
+  <div class="grid-width-8"></div>
+  <div class="grid-width-4"></div>
+</div>
+```
+
+## Flex layout
+The flex layout has no max width. It is composed of parent `row`s and child `col`s. Each row of columns needs to be enclosed in a `row` and **columns don't have to add up to anything**. If you're using a pure flex `-f{#]` grid, the toal number of columns is simply the sum of the `-f{#}`s. (you can mix and match with `col-fit` and `col-fill` as well, _or_ have underfilled rows using fixed or percentage width utilities). Columns may include nested rows and columns as in the following:
+
+```html
+<div class="row">
```

Comment:
I'm still thinking about this but am a little unsure of it. I mentioned I didn't like `f` from a naming perspective. I was thinking: what if we just went with `col`? Then if down the road we ditch flexbox for the newer and greater thing, we don't have to rename our classes?